src/ResourceManager/AzureBatch/Commands.Batch/Locations/GetBatchLocationQuotasCommand.cs

```diff
@@ -18,8 +18,8 @@
 
 namespace Microsoft.Azure.Commands.Batch
 {
-    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchSubscriptionQuotas), OutputType(typeof(PSBatchSubscriptionQuotas))]
-    public class GetBatchSubscriptionQuotasCommand : BatchCmdletBase
+    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchLocationQuotas), OutputType(typeof(PSBatchLocationQuotas))]
```

Comment:
@jasper-schneider The policy is to warn the user of impending breaking changes at least a sprint before making the breaking change.  The advice is to mitigate breaks with custom powershell code, not to remove properties from ourput types or from existing parameter sets.